cpu/samd5x/Kconfig

```diff
@@ -5,4 +5,34 @@
 # directory for more details.
 #
 
+config CPU_FAM_SAMD5X
+    bool
+    select CPU_COMMON_SAM0
+    select CPU_CORE_CORTEX_M4F
+    select HAS_BACKUP_RAM
+    select HAS_CORTEXM_MPU
+    select HAS_CPU_SAMD5X
+    select HAS_PERIPH_HWRNG
+
+## CPU Models
+config CPU_MODEL_SAME54P20A
+    bool
+    select CPU_FAM_SAMD5X
+
+## Declaration of specific features
+config HAS_CPU_SAMD5X
+    bool
+    help
+        Indicates that a 'samd5x' cpu is being used.
+
+## CPU common symbols
+config CPU_FAM
+    default "samd5x" if CPU_FAM_SAMD5X
```

Comment:
In #14485 you split that up into `CPU_FAM_SAML10` and `CPU_FAM_SAML11`.
Should we have at least `CPU_FAM_SAMD51` and `CPU_FAM_SAME54` here?

Reply:
In #14485 I followed the existing families. If it makes sense I can do the same here, but it should also be changed in the Makefile.features.